controllers/patch-controllers.js

```diff
@@ -0,0 +1,14 @@
+const { updateArticle } = require("../models/patch-models");
+
+exports.patchArticle = (req, res, next) => {
+  const { inc_votes } = req.body;
+  const { article_id } = req.params;
+
+  updateArticle(inc_votes, article_id)
+    .then((updatedArticle) => {
+      res.status(200).send(updatedArticle);
```

Comment:
you should send updatedArticle within an object - then you can change your test to find the article on a suitable key rather than just on body. I promise in front end you will appreciate it!